lib/grpcServer/handlers/platform/getDataContractHandlerFactory.js

```diff
@@ -0,0 +1,49 @@
+const {
+  server: {
+    error: {
+      InvalidArgumentGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+const {
+  GetDataContractResponse,
+} = require('@dashevo/dapi-grpc');
+
+/**
+ *
+ * @param {DriveAdapter} driveAPI
+ * @param {DashPlatformProtocol} dpp
+ * @returns {getDataContractHandler}
+ */
+function getDataContractHandlerFactory(driveAPI, dpp) {
+  /**
+   * @typedef getDataContractHandler
+   * @param {Object} call
+   * @returns {Promise<GetDocumentsResponse>}
+   */
+  async function getDataContractHandler(call) {
+    const { request } = call;
+    const id = request.getId();
+
+    if (id === null) {
+      throw new InvalidArgumentGrpcError('id is not specified');
+    }
+
+    const dataContractJSON = await driveAPI.fetchContract(id);
```

Comment:
It might throw RPCError with Invalid Param code so we need to catch and wrap it into InvalidArgumentGrpcError.